File: app/src/main/java/valiev/timur/tapfight/StartGameActivity.java
package valiev.timur.tapfight;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import valiev.timur.tapfight.game.GameActivity;

public class StartGameActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_start_game);
    }

    public void startGame(View view) {
        startActivity(new Intent(this, GameActivity.class));
    }
}

File: app/src/main/java/minteemer/tapfight/presentation/startGame/StartGameActivity.kt
package minteemer.tapfight.presentation.startGame

import android.content.Context
import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import minteemer.tapfight.databinding.ActivityStartGameBinding
import minteemer.tapfight.presentation.bubbles.BubblesGameActivity
import minteemer.tapfight.util.extensions.viewbinding.viewBinding

class StartGameActivity : AppCompatActivity() {

    private val binding: ActivityStartGameBinding by viewBinding(ActivityStartGameBinding::inflate)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding.startGameButton.setOnClickListener {
            startActivity(BubblesGameActivity.createIntent(this))
        }
    }

    companion object {

        fun createIntent(context: Context) = Intent(context, StartGameActivity::class.java)
    }
}
